src/ConstructiveSolidGeometry/LinePrimitives/Line.jl

```diff
@@ -29,6 +29,6 @@ distance(pt::CartesianPoint, l::Line) = norm((pt - l.origin) × l.direction) / n
 
 function _transform_into_object_coordinate_system(l::Line{T}, p::AbstractPrimitive) where {T}
     origin = _transform_into_object_coordinate_system(l.origin, p) 
-    direction = inv(rotation(p)) * l.direction
-    Line( origin, CartesianVector(direction) )  
+    direction = _transform_into_object_coordinate_system(l.direction, p)
+    Line( origin, direction )  
```

Comment:
This also allows us to replace some pieces of the old code with this new methods
(here: `l.direction` is already a `CartesianVector`)